run_gmsh/demo_create_gmsh_mesh.py

```diff
@@ -35,9 +27,31 @@
 mesh_output_dir = "./"
 
 
+# Extract the points, mesh sizes and name labels from the dictionary
+input_points_list = []
+mesh_size_list = []
+name_label_list = []
+for value in input_dict.values():
```

Comment:
warning should not be there in the demo file, as the demo file is intended to be as simple as possible, for the user to recreate. Warning and errors should be but in the set_mesh_size_function